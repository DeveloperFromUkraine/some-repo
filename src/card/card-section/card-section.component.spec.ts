import { ComponentFixture, TestBed } from "@angular/core/testing";
import { CardSectionComponent } from "index";
import { ComponentTest } from '../../../test/test-bed/component';

describe('CardSectionComponent', () => {
    let fixture: ComponentFixture<CardSectionComponent>;

    beforeEach(async () => {
        await ComponentTest.createTestBed([], [CardSectionComponent]);
    });

    beforeEach(() => {
        fixture = TestBed.createComponent(CardSectionComponent);
    });

    it('should match snapshot', () => {
        expect(fixture).toMatchSnapshot();
    }); 
});