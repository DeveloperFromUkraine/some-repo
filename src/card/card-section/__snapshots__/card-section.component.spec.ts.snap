// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardSectionComponent should match snapshot 1`] = `
<ign-card-section>
  <div
    class="card-section"
  >
    
  </div>
</ign-card-section>
`;
